fiftyone/utils/eval/coco.py

```diff
@@ -81,188 +87,212 @@ def evaluate_detections(
+                                        "iou": best_match_iou,
+                                    }
+                                    pred[pred_key]["matches"][iou_str] = {
+                                        "gt_id": best_match,
+                                        "iou": best_match_iou,
+                                    }
+                                    true_pos += 1
                                 else:
-                                    iscrowd = False
-
-                                # Cannot match two preds to the same gt unless
-                                # the gt is a crowd
-                                if curr_gt_match > -1 and not iscrowd:
-                                    continue
-
-                                # Ignore gts with an IoU lower than what was
-                                # already found
-                                if iou < best_match_iou:
-                                    continue
-
-                                best_match_iou = iou
-                                best_match = gt_id
-
-                            if best_match > -1:
-                                # If the prediction was matched, store the eval
-                                # id of the pred in the gt and of the gt in the
-                                # pred
-                                gt_to_store = gt_by_id[best_match][gt_key]
-                                gt_to_store["matches"][iou_str] = {
-                                    "pred_id": pred[pred_key]["pred_id"],
-                                    "iou": best_match_iou,
-                                }
-                                pred[pred_key]["matches"][iou_str] = {
-                                    "gt_id": best_match,
-                                    "iou": best_match_iou,
-                                }
-                                true_positives += 1
-                            else:
-                                false_positives += 1
-
-                        elif pred.label == cat:
-                            false_positives += 1
-
-                result_dict["true_positives"][iou_str] = true_positives
-                result_dict["false_positives"][iou_str] = false_positives
-                false_negatives = len(
-                    [
-                        g
-                        for g in dets["gts"]
-                        if g[gt_key]["matches"][iou_str]["pred_id"] == -1
-                    ]
-                )
-
-                result_dict["false_negatives"][iou_str] = false_negatives
-
-                if iou_str == save_iou_str:
-                    sample["tp_iou_%s" % save_iou_str] = true_positives
-                    sample["fp_iou_%s" % save_iou_str] = false_positives
-                    sample["fn_iou_%s" % save_iou_str] = false_negatives
-
-            sample[pred_field][pred_key] = result_dict
-
-            # @todo compute sample-wise AP
```

Comment:
Go ahead and trash `save_tp_fp_fn_counts()` now that `evaluate_detections()` only takes a single IoU